roles/scm/git/handlers/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: Remove SSH keys
```

Comment:
this is doing more than just removing SSH keys, perhaps change to something like "Clean-up temporary SSH directory"?